airflow/contrib/operators/gcs_to_gcs_transfer_operator.py

```diff
@@ -16,112 +16,10 @@
-    :type schedule: dict
-    :param object_conditions: Optional transfer service object conditions; see
-        https://cloud.google.com/storage-transfer/docs/reference/rest/v1/TransferSpec#ObjectConditions
-    :type object_conditions: dict
-    :param transfer_options: Optional transfer service transfer options; see
-        https://cloud.google.com/storage-transfer/docs/reference/rest/v1/TransferSpec#TransferOptions
-    :type transfer_options: dict
-    :param wait: Wait for transfer to finish; defaults to `True`
-    :type wait: bool
-
-    **Example**:
-
-    .. code-block:: python
-
-       gcs_to_gcs_transfer_op = GoogleCloudStorageToGoogleCloudStorageTransferOperator(
-            task_id='gcs_to_gcs_transfer_example',
-            source_bucket='my-source-bucket',
-            destination_bucket='my-destination-bucket',
-            project_id='my-gcp-project',
-            dag=my_dag)
-    """
-
-    template_fields = ('source_bucket', 'destination_bucket', 'description', 'object_conditions')
-    ui_color = '#e09411'
-
-    @apply_defaults
-    def __init__(self,
-                 source_bucket,
-                 destination_bucket,
-                 project_id=None,
-                 gcp_conn_id='google_cloud_default',
-                 delegate_to=None,
-                 description=None,
-                 schedule=None,
-                 object_conditions=None,
-                 transfer_options=None,
-                 wait=True,
-                 *args,
-                 **kwargs):
-
-        super(GoogleCloudStorageToGoogleCloudStorageTransferOperator, self).__init__(
-            *args,
-            **kwargs)
-        self.source_bucket = source_bucket
-        self.destination_bucket = destination_bucket
-        self.project_id = project_id
-        self.gcp_conn_id = gcp_conn_id
-        self.delegate_to = delegate_to
-        self.description = description
-        self.schedule = schedule
-        self.object_conditions = object_conditions or {}
-        self.transfer_options = transfer_options or {}
-        self.wait = wait
-
-    def execute(self, context):
-        transfer_hook = GCPTransferServiceHook(
-            gcp_conn_id=self.gcp_conn_id,
-            delegate_to=self.delegate_to)
-
-        job = transfer_hook.create_transfer_job(
-            project_id=self.project_id,
-            description=self.description,
-            schedule=self.schedule,
-            transfer_spec={
-                'gcsDataSource': {
-                    'bucketName': self.source_bucket,
-                },
-                'gcsDataSink': {
-                    'bucketName': self.destination_bucket,
-                },
-                'objectConditions': self.object_conditions,
-                'transferOptions': self.transfer_options,
-            }
-        )
-
-        if self.wait:
-            transfer_hook.wait_for_transfer_job(job)
+warnings.warn("This module is deprecated. You should use "
```

Comment:
To jest dyskusyjna zmiana. Możliwe, że będziemy musieli cofnąć te zmianę. Nie miałem jeszcze czasu, aby wypracować w community standardów w jaki sposób to zrobić prawidłowo, ale myślę, że K.l to zaakceptuje, bo popierał moje rozwiązanie.  http://ujeb.se/CtD5U (przez skracarkę, aby nie robić referencji)

Reply:
W takim razie zostawiam